libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/DualLocationMocker.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.navigation.testing.ui
+
+import android.location.Location
+
+/**
+ * Mocker that pushes updates to both location provider implementations
+ * so that the sample is available regardless where the running location engine tries to get it from.
```

Comment:
Hm, I though we would use only fused provider if the library is available. Judging by the ticket description it's not the case.

Reply:
It is the case, however, there's a bug in the Common SDK that only delivers last location from the Android provider right now, see CORESDK-1528.